addon/components/scatter-chart.js

```diff
@@ -310,15 +310,19 @@ const ScatterChartComponent = ChartComponent.extend(LegendMixin, FloatingTooltip
       d3.select(element).classed('hovered', true);
       var formatXValue = this.get('formatXValue');
       var formatYValue = this.get('formatYValue');
-      var xValueDisplayName = $("<br /><span class=\"name\" />").text(this.get('xValueDisplayName') + ": ");
-      var yValueDisplayName = $("<span class=\"name\" />").text(this.get('yValueDisplayName') + ": ");
-      var content = $("<span class=\"tip-label\" />").text(data.group)
+      var xValueDisplayName = $('<span class="name" />').text(this.get('xValueDisplayName') + ': ');
+      var yValueDisplayName = $('<span class="name" />').text(this.get('yValueDisplayName') + ': ');
+      var xValue = $('<span class="value" />').text(formatXValue(data.xValue));
```

Comment:
I think I made the mistake of assuming `formatXValue` would always return `NaN`, I didn't think that people might extend it.